packages/p2p-v2/src/components/PaymentMethodsFormFooter/PaymentMethodsFormFooter.scss

```diff
@@ -1,16 +1,16 @@
-.p2p-v2-payment-method-form {
-    &__buttons {
-        display: flex;
-        justify-content: end;
-        gap: 1.6rem;
+.p2p-v2-payment-methods-form-footer {
+    display: flex;
+    justify-content: end;
+    gap: 0.8rem;
+    background: #fff;
+    margin-top: 2rem;
 
-        &--cancel {
-            background: transparent;
-            border: 2px solid #999999;
-
-            &:active:not([disabled]) {
-                border: 1px solid #999999;
-            }
-        }
+    @include mobile {
+        margin-top: 0;
+        width: 100%;
+        padding: 1.6rem 2.4rem;
```

Comment:
I've seen both inline styles and scss classes being used, is there a best practice that we want to follow here?

Reply:
for p2p-v2, we usually use tailwind if its relatively short, but if its a bit long and you're using too much responsive checks like lg:px-0 px-2 we create a scss class for it.